R/gatherIAMCTimeseries.R

```diff
@@ -0,0 +1,62 @@
+#' gatherIAMCTimeseries
+#'
+#' Pivots periods columns (as they occur in IAMC wide format/timeseries) to the typical `quitte` long list format
+#'
+#' @md
+#' @param df `quitte` data frame in IAMC wide format (i.e. years/periods are columns)
+#' @param quitte_case In IAMC case columns first letters are capitalized, quitte case is lower case. Default is FALSE
+#' @return `quitte` data frame in long IAMC (= `quitte`) format
```

Comment:
The function does not return an object of class "quitte", which users might expect.